### ISSUE TITLE
[14.0, 15.0, ?] build broken - facturx

### ISSUE THREAD

**Issue by simahawk** [OCA/edi#1084]
```
2024-12-03 15:29:44,540 2210 ERROR odoo odoo.tests.loader: Can not `import account_invoice_facturx`. 
Traceback (most recent call last):
  File "/opt/odoo/odoo/tests/loader.py", line 29, in _get_tests_modules
    mod = importlib.import_module('.tests', modpath)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/__w/edi/edi/account_invoice_facturx/tests/__init__.py", line 1, in <module>
    from . import test_facturx_invoice
  File "/__w/edi/edi/account_invoice_facturx/tests/test_facturx_invoice.py", line 7, in <module>
    from facturx import get_facturx_level
  File "/opt/odoo-venv/lib/python3.6/site-packages/facturx/__init__.py", line 1, in <module>
    from .facturx import generate_from_file, \
  File "/opt/odoo-venv/lib/python3.6/site-packages/facturx/facturx.py", line 37, in <module>
    import importlib.resources
ModuleNotFoundError: No module named 'importlib.resources'
```

**Comment by simahawk** [comment]
Issue found here https://github.com/akretion/factur-x/issues/48

We have to pin https://pypi.org/project/factur-x/3.1

**Comment by simahawk** [comment]
Auto-merged by GH but 15.0 still pending
